pkgs/development/python-modules/scikit-rf/default.nix

```diff
@@ -99,6 +102,10 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  # test_calibration.py generates a divide by zero error on darwin
+  disabledTestPaths =
+    lib.optional stdenv.isDarwin "skrf/calibration/tests/test_calibration.py";
```

Comment:
Could you create and link to an issue upstream?

Reply:
hmm. if it were linux yes, but for macos, i do not have access to a system so i can not.

Reply:
can mark it as broken tho.

Reply:
Pasting the error from ofborg into an issue could still be helpful for upstream. If upstreams determines this to affect correctness then i'd mark it broken.

Reply:
> Pasting the error from ofborg into an issue could still be helpful for upstream, but do let them know that you (nor i) can easily aid them given the lack of a mac. If upstreams determines this to affect correctness then i'd mark it broken.

perhaps but that is below the quality of a bug report that i am comfortable with sending.

Reply:
looking at the test it seems like the random matrix they are trying to calculate the determinate of is non-invertible but without access to the system to know what libs are on it or hardware i don't have much more to offer.

Reply:
posted issue: https://github.com/scikit-rf/scikit-rf/issues/972

Reply:
I'm confused how this is failing. if i am reading the log correctly -- and the matrix passed to det matches when i add an assert on x64 linux -- it seems like this 2x2 matrix is triggering the issue.

```
array([[5.e-08+0.j, 1.e+00+0.j],
       [1.e+00+0.j, 5.e-08+0.j]])
```
and it's determinate is the difference of the product of the diagonals (ad-bc). feel like i'm missing something as i haven't done any complex linear algebra. anyway, think that darwin aarch64 should be marked broken.

Reply:
tho looking at the numpy [code](https://github.com/numpy/numpy/blob/391554abee3c633a93fbcd47df9a345f5aedb148/numpy/linalg/umath_linalg.cpp#L1248) it is performing an LU decomp and taking logs. i'd have to spend some time looking at this more to understand it better.

Reply:
ok -- a kind person ran the test code on macos:

```py
#! /usr/bin/env nix-shell
#! nix-shell -i python3 -p "python3.withPackages(p: [p.numpy])"
import numpy as np
print(np.__version__)
a = np.array([[5.e-8+0.j, 1+0.j], [1+0.j, 5.e-8+0.j]])
print(a)
print(np.linalg.det(a))
```

```console
./test.py
1.25.2
[[5.e-08+0.j 1.e+00+0.j]
 [1.e+00+0.j 5.e-08+0.j]]
/nix/store/rpdx49knxarbr4lmkwfi5l0461lyp6bd-python3-3.11.6-env/lib/python3.11/site-packages/numpy/linalg/linalg.py:2180: RuntimeWarning: divide by zero encountered in det
  r = _umath_linalg.det(a, signature=signature)
/nix/store/rpdx49knxarbr4lmkwfi5l0461lyp6bd-python3-3.11.6-env/lib/python3.11/site-packages/numpy/linalg/linalg.py:2180: RuntimeWarning: invalid value encountered in det
  r = _umath_linalg.det(a, signature=signature)
```

so the failure seems to be unrelated to to scikit-rt

Reply:
You went above and beyond, exemplary